### PR TITLE
Actualization of the script for 1921

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
ATF Test Scripts to fix #[2333](https://github.com/smartdevicelink/sdl_atf_test_scripts/issues/2333)

This PR is **ready** for review.

### Summary
Actualization of the script for 1921

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua [hunk 1]
```diff
@@ -125,18 +147,16 @@ local function contains(pTbl, pValue)
...
---! OnPermissionsChange notification
+--[[ @CheckCuttingUnknowValues: expectation of OnPermissionsChange notification and check its content
 --! @parameters:
+--! pNotifTimes - expected number of OnPermissionsChange notification
 --! self - test object
 --! @return: none
 --]]
-local function CheckCuttingUnknowValues(ptuUpdateFunc, self)
-  commonDefects.rai_ptu_n_without_OnPermissionsChange(1, ptuUpdateFunc, self)
+local function CheckCuttingUnknowValues(pNotifTimes, self)
```
Comment (jacobkeeler) [inline test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua:156]:
```suggestion
local function CheckCuttingUnknownValues(pNotifTimes, self)
```

File: test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua [hunk 2]
```diff
@@ -231,25 +285,38 @@ local function DisallowedRPC(RPC, params, interface, self)
...
+runner.Step("PTU with unknown parameters", ptu, { ptuUpdateFuncParams })
 runner.Step("Check applying of PT by processing GetVehicleData", SuccessfulProcessingRPC,
-  { "GetVehicleData", { gps = true }, "VehicleInfo" })
+  { "GetVehicleData", { gps = true }, "VehicleInfo", { gps = gpsDataResponse } })
 runner.Step("Check applying of PT by processing SubscribeVehicleData", DisallowedRPC,
   { "SubscribeVehicleData", { gps = true }, "VehicleInfo" })
 
 runner.Step("Remove Snapshot and trigger PTU, check new created PTS", removeSnapshotAndTriggerPTUFromHMI)
-runner.Step("Invalid_PTU_after_cutting_of_unknown_values", commonDefects.unsuccessfulPTU, { ptuUpdateFuncNotValid })
+runner.Step("Invalid_PTU_after_cutting_of_unknown_values", commonDefects.unsuccessfulPTU,
```
Comment (jacobkeeler) [inline test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua:318]:
```suggestion
runner.Step("Invalid_PTU_after_cutting_off_unknown_values", commonDefects.unsuccessfulPTU,
```

File: test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua (hunk 1 above)
Comment (dboltovskyi) [reply test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua]: @jacobkeeler Please find update in d45c6c1

File: test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua (hunk 2 above)
Comment (dboltovskyi) [reply test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua]: @jacobkeeler Please find update in d45c6c1

**Pull request merged**